src/components/SearchBar/SearchBar.js

```diff
@@ -57,19 +53,19 @@ class SearchBar extends React.Component {
   render() {
     return (
       <div className="SearchBar">
-  <div className="SearchBar-sort-options">
-    <ul>
-      {this.renderSortByOptions()}
-    </ul>
-  </div>
-  <div className="SearchBar-fields">
-    <input placeholder="Search Businesses" onChange={this.handleTermChange} onChange={this.handleLocationChange}  />
-    <input placeholder="Where?" onChange={this.handleTermChange} onChange={this.handleLocationChange} />
```

Comment:
There was a double declaration of  `onChange` here which was causing the confusion. I just removed the final one and then bound the corresponding event